app/Http/Controllers/BillingController.php

```diff
@@ -724,42 +724,54 @@ private function generatePaymentMethodListForPaymentPage(): array
     {
         $paymentMethods = [];
         $validAccountMethods = $this->getPaymentMethods();
+
         foreach ($validAccountMethods as $validAccountMethod) {
             if (
                 $validAccountMethod->type == 'credit card'
                 && config('customer_portal.enable_credit_card_payments') == 1
             ) {
-                $paymentMethods[$validAccountMethod->id] = utrans(
-                    'billing.payUsingExistingCard',
-                    [
-                        'card' => '****'.$validAccountMethod->identifier.' ('
-                            .sprintf('%02d', $validAccountMethod->expiration_month).' / '
-                            .$validAccountMethod->expiration_year.')'
-                    ]
-                );
+                $paymentMethods[$validAccountMethod->id . '_credit_card'] = [
```

Comment:
This fixes this issue from a different PR 
https://dev.azure.com/sonarsoftware/Sonar/_workitems/edit/9390